README.md

```diff
@@ -21,6 +21,11 @@ Requirements:
 Setup:
 - `npm install`
 
+Setup (Windows):
```

Comment:
For next time include unrelated changes to the fix in a different pull request. That makes them easier to understand.